runtime/cudaq/qis/qubit_qis.h

```diff
@@ -990,4 +573,38 @@ std::vector<T> slice_vector(std::vector<T> &original, std::size_t start,
   return ret;
 }
 
-} // namespace cudaq
\ No newline at end of file
+} // namespace cudaq
+
+#define __qop__ __attribute__((annotate("user_custom_quantum_operation")))
+
+/// Register a new custom unitary operation providing a unique name,
+/// the number of target qubits, the number of rotation parameters (can be 0),
+/// and the unitary matrix as a 1D row-major `std::vector<complex>`
+/// representation following a MSB qubit ordering.
```

Comment:
Curious... why require MSB when the internal default is LSB?

Reply:
I'm open to this part changing. I went with that since I'm most familiar with the QC textbook ordering as opposed to the physicist ordering (e.g. qiskit).